src/constants/shared/helpers.js

```diff
@@ -9,3 +9,7 @@ export function limitNumericalInput(input, key, value, min, max, integer) {
     input[key] = integer ? Math.floor(Number(value)) : Number(value);
   }
 }
+
+export function limitNumberOfDecimals(number) {
```

Comment:
In the future it would be great to add one more param, which will define how many decimals we want to use...